aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -73,6 +74,10 @@ module aptos_framework::stake {
     const EVOTING_POWER_INCREASE_EXCEEDS_LIMIT: u64 = 19;
     /// Stake pool does not exist at the provided pool address.
     const ESTAKE_POOL_DOES_NOT_EXIST: u64 = 20;
+    /// Fullnode addresses are not valid.
+    const EINVALID_FULLNODE_ADDRESSES: u64 = 21;
+    /// Network addresses are not valid.
+    const EINVALID_NETWORK_ADDRESSES: u64 = 22;
```

Comment:
I wonder if we can make this clearer, like EINVALID_VALIDATOR_ADDRESSES. Since this is for validators, and the other is for VFNs right? Though I know you're just matching what the fields are called now.

Reply:
@gregnazario @sherry-x Wdyt?

Reply:
I'd vote for invalid_validator_address

Reply:
Updated